### PR TITLE
feat: Five Minutes to Feature Flags Tutorial

### PR DESCRIPTION
**Pull request by agardnerIT**
## This PR
Adds a new tutorial "Five Minutes to Feature Flags".

As discussed on Slack, this mirrors @moredip's blog post: `https://blog.thepete.net/blog/2023/03/02/five-minutes-to-feature-flags/`

### CODE REVIEW

**Comment by agardnerIT** [comment]
From @moredip: "I'd like to hold off on releasing it under the OF name until we're done moving that provider of mine under the OF contrib umbrella (https://github.com/open-feature/js-sdk-contrib/pull/252)."

**Comment by agardnerIT** [comment]
@moredip @beeme1mr this should be ready for review now as it uses Pete's new copy and the minimal provider. Thanks!

**Review by moredip — APPROVED** [review]
:shipit:

File: five-minutes-to-feature-flags/05-more-providers.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { LaunchDarklyProvider } from '@launchdarkly/openfeature-node-server';
+
+const ldClient = init('[YOUR-SDK-KEY]');
+await ldClient.waitForInitialization();
+OpenFeature.setProvider(new LaunchDarklyProvider(ldClient));
+```
+
+### flagd
+```
+OpenFeature.setProvider(new FlagdProvider({
```
Comment (beeme1mr) [inline five-minutes-to-feature-flags/05-more-providers.md:21]:
```suggestion
import { FlagdProvider } from '@openfeature/flagd-provider';

OpenFeature.setProvider(new FlagdProvider({
```

**Review by beeme1mr — APPROVED** [review]
Nice :shipit:

**Pull request merged**